guides/hack/03-expressions-and-operators/75-comparisons.md

```diff
@@ -30,7 +30,7 @@ Arrays are compared by length, then by lexicographic ordering on
 elements:
 
 ``` Hack
-varray[5] < varray[1, 2]   // true (LHS has fewer elements)
+vec[5] < vec[1, 2]   // true (left-hand side has fewer elements)
```

Comment:
Nice, I like the reduction in jargon :)

Reply:
thanks! a lot of instances of LHS which took me forever to realize was slang...